threescale_api/defaults.py

```diff
@@ -178,6 +178,8 @@ def read_by_name(self, name: str, **kwargs) -> 'DefaultResource':
         for item in self._list(**kwargs):
             if item.entity_name and item.entity_name == name:
                 return item
+            if item.get("name") == name:
```

Comment:
I know in description of function is that it searches in system_name, name and org_name, but I don't recommend to change this and I prefer to fix account plan definition.